src/loki/isochrone_action.cc

```diff
@@ -31,41 +31,43 @@ namespace valhalla {
         throw valhalla_exception_t{400, 152, std::to_string(max_contours)};
       size_t prev = 0;
       for(const auto& contour : *contours) {
-        const int c = GetOptionalFromRapidJson<int>(contour, "/time").get_value_or(-1);
-        if(c < prev || c == -1)
-          throw valhalla_exception_t{400, 111};
-        if(c > max_time)
-          throw valhalla_exception_t{400, 151, std::to_string(max_time)};
-        prev = c;
+        if (contour.HasMember("time")) {
+          const int c = contour["time"].IsFloat() ?  GetOptionalFromRapidJson<float>(contour, "/time").get_value_or(-1) : GetOptionalFromRapidJson<int>(contour, "/time").get_value_or(-1);
```

Comment:
i would just do like you were doing with boost get optional and just always get `float`, dont check for the member and dont check if its float. basically keep the code the same but change int to float in the template paramter